homeassistant/components/fints/sensor.py

```diff
@@ -168,7 +168,8 @@ def is_balance_account(self, account: SEPAAccount) -> bool:
         if not account_information:
             return False
 
-        if 1 <= account_information["type"] <= 9:
+        account_type = account_information.get("type")
+        if isinstance(account_type, int) and 1 <= account_type <= 9:
```

Comment:
Would it be sufficient to say `account_type is not None` or even `account_type` (assuming `account_type` cannot be 0) rather than using `isinstance`? Can `account_type` be anything other than `None` or `int`?

Reply:
Consider: 
```
        if account_type := account_information.get("type"):
            return 1 <= account_type <= 9:
````

Reply:
Really like your idea. I always forget we have a walrus operator now ;)